ironfish-cli/package.json

```diff
@@ -19,7 +19,8 @@
     "oclif": "1.16.1",
     "tsc-watch": "4.2.9",
     "typescript": "4.3.4",
-    "yarn": "^1.22.10"
+    "yarn": "^1.22.10",
+    "rimraf": "^3.0.2"
```

Comment:
Last change, could you alphabetize this import?

Reply:
Yup, done